### PR TITLE
PAPP-24020: Convert app to community app

### PR DESCRIPTION
**Pull request by ebhoir-crest**
Please ensure your pull request (PR) adheres to the following guidelines:

- Please refer to our contributing documentation for any questions on submitting a pull request, link: [Contribution Guide](https://github.com/splunk-soar-connectors/.github/blob/main/.github/CONTRIBUTING.md)

#### Please check if your PR fulfills the following requirements:
- [ ] Testing of all the changes has been performed (for bug fixes / features)
- [ ] The readme.html has been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Use the following format for the PR description: `<App Name>: <PR Type> - <PR Description>`
- [ ] Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.
- [ ] Verify all checks are passing.
- [ ] Do NOT use the `next` branch of the forked repo. Create separate feature branch for raising the PR.
- [ ] Do NOT submit updates to dependencies unless it fixes an issue.
#### Please check the type of change your PR introduces:
- [ ] New App
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation
- [ ] Other (please describe): 

## Release Notes (REQUIRED)
- Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.

## What is the current behavior? (OPTIONAL)
- Describe the current behavior that you are modifying.

## What is the new behavior? (OPTIONAL)
- Describe the behavior or changes that are being added by this PR.

## Other information (OPTIONAL)
- Any other information that is important to this PR such as screenshots of how the component looks before and after the change.

## Pay close attention to (OPTIONAL)
- Any specific code change or test case points which must be addressed/reviewed at the time of GA release.

## Screenshots (if relevant)

---
Thanks for contributing!

### CODE REVIEW

File: ciscoasa.json [hunk 1]
```diff
@@ -3,22 +3,20 @@
...
     "package_name": "phantom_ciscoasa",
     "product_vendor": "Cisco Systems",
     "product_name": "Cisco ASA",
     "product_version_regex": ".*",
-    "min_phantom_version": "4.9.39220",
-    "latest_tested_versions": [
-        "On Prem, Version 7.6"
-    ],
+    "min_phantom_version": "5.1.0",
+    "fips_compliant": true,
```
Comment (dgopani-crest) [inline ciscoasa.json]: remove this or mark as a false https://splunk.slack.com/archives/GBSGHUUD9/p1643701225287839?thread_ts=1639635017.100200&cid=GBSGHUUD9

File: ciscoasa.json [hunk 2]
```diff
@@ -3,22 +3,20 @@
     "name": "Cisco ASA",
     "description": "This app supports containment actions in addition to investigative actions on a Cisco ASA device",
     "type": "firewall",
-    "publisher": "Splunk",
+    "publisher": "Splunk Community",
     "main_module": "ciscoasa_connector.py",
     "app_version": "2.0.4",
```
Comment (dgopani-crest) [inline ciscoasa.json]: 3.0.0

**Comment by ebhoir-crest** [comment]
Static test will pass once approved_modules.json MR: [!1777](https://cd.splunkdev.com/phantom/app_release/-/merge_requests/1777) is merged.

File: ciscoasa.json [hunk 3]
```diff
@@ -3,22 +3,20 @@
...
-    "latest_tested_versions": [
-        "On Prem, Version 7.6"
-    ],
+    "min_phantom_version": "5.1.0",
+    "fips_compliant": false,
     "python_version": "3",
     "logo": "logo_cisco.svg",
     "logo_dark": "logo_cisco_dark.svg",
-    "license": "Copyright (c) 2014-2021 Splunk Inc.",
+    "license": "Copyright (c) 2014-2022 Splunk Inc.",
```
Comment (paulnawat) [inline ciscoasa.json:19]: For reference, it should be sufficient to update the license year here. Additional job will be run to use the year here to update the license years of other parts.

**Review by paulnawat — APPROVED** [review]

File: ciscoasa.json (hunk 3 above)
Comment (ebhoir-crest) [reply ciscoasa.json:19]: Thanks @paulnawat I will follow this process hence forth

**Comment by ebhoir-crest** [comment]
As the app is to be converted to community supported app, there are no test playbooks available in repo which results is static test failure.

**Pull request merged**